apps/worker/src/publisher/publisher.module.ts

```diff
@@ -51,17 +51,30 @@ import { QueueConstants } from '../../../../libs/common/src';
       },
       inject: [ConfigService],
     }),
-    BullModule.registerQueue({
-      name: QueueConstants.PUBLISH_QUEUE_NAME,
-      defaultJobOptions: {
-        attempts: 1,
-        backoff: {
-          type: 'exponential',
+    BullModule.registerQueue(
+      {
+        name: QueueConstants.PUBLISH_QUEUE_NAME,
+        defaultJobOptions: {
+          attempts: 1,
+          backoff: {
+            type: 'exponential',
+          },
+          removeOnComplete: false,
+          removeOnFail: false,
```

Comment:
We may not want to clear the publishQueue and let status monitoring decide if that job is good to be out or need a re-queue based on any issue with transaction